server_manager/model/server.ts

```diff
@@ -186,10 +186,31 @@ export interface AccessKey {
   dataLimit?: DataLimit;
 }
 
-export type BytesByAccessKey = Map<AccessKeyId, number>;
-
 // Data transfer allowance, measured in bytes.
 // NOTE: Must be kept in sync with the definition in src/shadowbox/access_key.ts.
 export interface DataLimit {
   readonly bytes: number;
 }
+
+export type ServerMetricsJson = {
```

Comment:
The domain model should not be tied to the serialization format. This shouldn't be named `Json`.
There should be a function somewhere converting from the JSON format to the model object.